README.md

```diff
@@ -58,6 +58,32 @@ log/database/PID files.
 files there. Files in the `etc/` directory of this repository can be used as an
 example (also see below for more about configuration)
 
+Installing automation-tools in Docker
+-------------------------------------
+
+Run each line below to install automation-tools in a Docker environment:
+
+```bash
+sudo docker exec -it compose_archivematica-dashboard_1 /bin/bash
+apt-get update
+mkdir /usr/lib/archivematica
+mkdir /usr/lib/archivematica/automation-tools
+cd /usr/lib/archivematica/automation-tools/
+git clone https://github.com/artefactual/automation-tools.git .
+mkdir /usr/share/python/automation-tools
+cd /usr/share/python/automation-tools
+virtualenv .env
+source .env/bin/activate
```

Comment:
Is this required? I use `.env` by default because a lot of other Python programmers use it, and the virtualenv docs use `venv`. Within Python it doesn't really matter what is used here, it's like personal preference / syntax choice. Does it matter within automation-tools for a reason I haven't deduced?

Reply:
Not really, but it's probably a good idea  to isolate the automation tools' dependencies when using a container that's dedicated to something else. 

FWIW I don't tend to use `venvs` on my local machine so I get the personal preference thing.